### ISSUE TITLE
Add : Savings Account Status (Active or Not) in SavingsAccount list item

### ISSUE THREAD

**Issue by therajanmaurya** [openMF/self-service-app#44]
In current user is able to see the SavingsAccount list but there is no sign to look. what is the status of SavingsAccount. it is Active or not.

How to achieve this issue : 
- Add a circular Image view in SavingsAccount Item list.
- show the status of the Saving Account.

**Comment by sagar15795** [comment]
@therajanmaurya I am working on this issue.

**Comment by sagar15795** [comment]
@therajanmaurya  Can you tell me what type of circular image we have to use? 
Is it symbolic icon images or text-based images ?

**Comment by therajanmaurya** [comment]
Right now we want this type of UI and forget about the colors, I will tell you color sequence soon.
![group 13](https://cloud.githubusercontent.com/assets/5724482/19827968/3bb2c922-9dd7-11e6-8869-91002ef82dd2.png)

Image View will be good and as accordingly we will fill color into them.

**Comment by therajanmaurya** [comment]
@sagar15795 Please look, how you have to proceed with this Issue. Here #45 5th comment.

**Comment by sagar15795** [comment]
Thanks @therajanmaurya.

Can you tell me what is the order of status attribute?

**Comment by therajanmaurya** [comment]
I think You didn't read the full comment.

**I will open separate issue for this setting correct color order for account in Adapter for both Savings and Loan.**

**Comment by therajanmaurya** [comment]
Color order means, there is many types of accounts :  Active, InActive, pendingApproval, waitingForDisbursal etc and for every type of accout we need to show his status by color indicator. 
but in this issue you have do one thing.

Add Circular ImageView or you can use VIew Class instead because we want to fill the color in the view. So Please Use View. For Example : http://stackoverflow.com/a/14209836/3176277.

Set any random color in Adapter that is already present in colors.xml. Do not need to add any exatra color.

I will open separate issue for this setting correct color order for account in Adapter for both Savings and Loan in which you have check the status of the issue and then show color accordingly.

**Comment by sagar15795** [comment]
I understand what are you saying but my question is different. I am asking about order of status attributes from the following attributes 

```
        "approved": false,
        "rejected": false,
        "withdrawnByApplicant": false,
        "active": false,
        "closed": false,
        "prematureClosed": false,
        "transferInProgress": false,
        "transferOnHold": false,
        "matured": false
```

So I have following questions 
1) At any particular time, only one attribute will be true or not 
2) If not then which attribute has high weightage.

**Comment by therajanmaurya** [comment]
Here is order 

```
active
approved
submittedAndPendingApproval
closed
```

It will be like this.

and Please try to talk about the issue on the Github issue, If you have any query that is not related to issue. you can ask on Gitter channel because we are using the issue as a tracker, So we can easily track the fixes If we needed.

**Comment by sagar15795** [comment]
@therajanmaurya I have created UI as suggested by you 

![image](https://cloud.githubusercontent.com/assets/5468111/19853470/666dcba8-9f8f-11e6-9cda-9a29ecb8e915.png)

Is it correct or not ?

**Comment by sagar15795** [comment]
@therajanmaurya  
I have also implemented using the same approach as used in #52. You suggested that change view to custom Circular ImageView. So, I have to wait till you merge #52. Otherwise, it may create conflict for the custom Circular ImageView class.

**Comment by therajanmaurya** [comment]
@sagar15795 It's great. send a PR, I will review and merge :)

**Comment by therajanmaurya** [comment]
Fixed in #53
